index.html

```diff
@@ -56,20 +57,22 @@ <h2>inclusive of your skin.</h2>
 			
 			<!-- These are the boxes containing the two images side by side -->
 			<div class="left">
+				<img src="images/skinquiz.jpg">
 				<h1>New to skincare?</h1>
 				<p>Find out your skin type here</p>
 				<a href="quiz.html">TAKE THE QUIZ</a>
 			</div>
 			
 			<div class="right">
+				<img src="images/skinconcern.jpeg">
 				<h1>Have a skin concern?</h1>
 				<p>Read more articles about common skin issues here</p>
 				<a href="article-list.html">SKINCARE ARTICLES</a>
 			</div>
 
 			<!-- This is the individual text box-->
 			<div class="subtitle">
-				<p>Browse our top articles</p>
+				<p>Browse our recommended articles</p>

```

Comment:
Reworded the subheading for the suggested articles at the bottom of the Index page.